packages/lesmis/src/components/Visuals/Chart.js

```diff
@@ -173,7 +182,13 @@ export const Chart = ({
   ) : (
     <>
       <VisualHeader name={name} isLoading={isFetchingInBackground}>
-        <Toggle onChange={handleTabChange} value={selectedTab} exclusive />
+        <GridContainer>
+          <Toggle onChange={handleTabChange} value={selectedTab} exclusive />
+          <FavouriteButton
+            favouriteStatus={favouriteStatus}
+            handleFavouriteStatusChange={handleFavouriteStatusChange}
+          />
```

Comment:
Hey I wanted to make a suggestion about the whole `favouriteStatus` thing. So as far as I understand, it seems we use the `IDLE` state just to refer to the how to treat the component when the user is not logged in?

It seems to me it could be simpler if we just handled that logic separately, eg.
```suggestion
          // Don't render FavouriteButton if user is not logged in
          userIsLoggedIn && <FavouriteButton
            favouriteStatus={favouriteStatus}
            handleFavouriteStatusChange={handleFavouriteStatusChange}
          />
```

Then we could revert the `favouriteStatus` to just being a simple boolean `isFavourite`, which I feel makes things a lot clearer. Thoughts?

Reply:
Yeah thanks for that, it is clearer! Will do the changes.